.github/workflows/api-tests.yml

```diff
@@ -39,3 +39,19 @@ jobs:
     - name: Run Tests
       run: |
         pytest --cov=api
+
+  flake8-lint:
+    runs-on: ubuntu-latest
+    name: Lint
+    steps:
+    - name: Check out source repository
+      uses: actions/checkout@v2
+    - name: Set up Python environment
+      uses: actions/setup-python@v2
+      with:
+        python-version: "3.8"
+    - name: flake8 Lint
+      uses: py-actions/flake8@v2
+      with:
+        max-line-length: "100"
+        path: "api"
```

Comment:
I am not sure how `path` will work here when the `working directory` is already "api". Once your PR is approved and checks are run. Try to make sure it runs `flake8` on root `api` directory instead of `api/api`